oelint_parser/cls_item.py

```diff
@@ -8,7 +8,7 @@
 from oelint_parser.rpl_regex import RegexRpl
 
 __safeline_split_regex__ = regex.compile(r"\s|\t|\x1b")
-__id_regex__ = regex.compile(r"[a-z0-9{}$]+")  # noqa: P103
+__id_regex__ = regex.compile(r"[a-zA-Z0-9{}$]+")  # noqa: P103
```

Comment:
Do you happen to have an example at hand?
Also this needs a test case

Reply:
Unfortunately this change breaks other cases, where variables contain underscores and I'm not sure if there is a good solution to that. I would have to fall back to manual parsing to fix my issue.
FYI, this happens in an internal repo, which I cannot show here.

Reply:
@zinzila can we close that one?
Btw I was not asking to reveal your private repos, but at least an example where upper case letter are needed

Reply:
For the record the example was: `SRCREV_RxCpp` 
And I actually failed to find any exact BitBake syntax reference, where this would be specified.
I will close the PR.